Sources/Controllers/Settings/OSMEditing/OAMappersViewController.m

```diff
@@ -378,6 +376,17 @@ - (NSString *) tableView:(UITableView *)tableView titleForFooterInSection:(NSInt
     return [_footers objectForKey:@(section)];
 }
 
+- (CGFloat)tableView:(UITableView *)tableView estimatedHeightForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    NSDictionary *item = [self getItem:indexPath];
+    if ([item[@"type"] isEqualToString:[OAMultiIconsDescCustomCell getCellIdentifier]])
+    {
+        return [item.allKeys containsObject:@"bottom_description"] ? 66. : 48;
+    }
+
+    return UITableViewAutomaticDimension;
```

Comment:
Estimated height cannot be UITableViewAutomaticDimension, it should be an approximate average value of all the cells on the screen (or the value of the most frequent cell)